authors/apps/authentication/views.py

```diff
@@ -44,7 +51,20 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
 
-        return Response(serializer.data, status=status.HTTP_200_OK)
+        token = generate_token(serializer.data).decode()
```

Comment:
@davenjeru That's a great PR, I just noticed that you haven't included the instructions on how to modify the .env file to add the secret key. This would help in testing out your work.

Reply:
The secret key is configured in [authors/settings.py](https://github.com/andela/ah-scorpion/blob/develop/authors/settings.py#L22).
The app should work without changing your `.env` file.
For security reasons, I'll set the Secret Key to be read from an environment variable before 9.30 am today.